app/pages/organization/stats/components/ProjectStats/components/Heading.jsx

```diff
@@ -0,0 +1,126 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { IndexLink } from 'react-router';
+import styled from 'styled-components';
+
+import Avatar from '../../../../../project/components/ProjectNavbar/components/Avatar';
+import getCompleteness from '../helpers/getCompleteness';
+
+const StyledHeading = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: .5em;
+`;
+
+const StyledAvatar = styled(Avatar)`
+  box-shadow: none;
+  margin-right: .5em;
+`;
+
+const StyledLink = styled(IndexLink).attrs({
+  activeClassName: 'active'
```

Comment:
Same comment here about using CSS to indicate selected state.

Reply:
Wasn't using, removed.